### ISSUE TITLE
Bug in JumpStatementsSpacing if there is comment line below ?

### ISSUE THREAD

**Issue by ncou** [slevomat/coding-standard#1093]
Hi,

I think i found a little bug with the JumpStatementsSpacing function. The comments line after the code doesn't seem to be ignored.

I use this configuration :

```
<rule ref="SlevomatCodingStandard.ControlStructures.JumpStatementsSpacing">
        <properties>
            <property name="tokensToCheck" type="array">
                <element value="T_RETURN" />
                <element value="T_THROW" />
                <element value="T_BREAK" />
                <element value="T_CONTINUE" />
                <element value="T_YIELD" />
                <element value="T_YIELD_FROM" />
            </property>
        </properties>
    </rule>
```

And my code is the following : 

```php
if ($plaintext === false) {
    // @codeCoverageIgnoreStart
    throw new RuntimeException('Encryption library: Decryption (symmetric) of content failed: ' . openssl_error_string());
    // @codeCoverageIgnoreEnd
}
```

When i check the code, there is an error :

`
ERROR [x] Expected 1 lines after "throw", found 0.
                    (SlevomatCodingStandard.ControlStructures.JumpStatementsSpacing.IncorrectLinesCountAfterControlStructure)
`

But the last line is a comment and should be ignored in the sniff. Or i am missing something ?

I think it could be the same use case as this ticket where the comments should be ignored (but this time it's if the comment is the last line).
https://github.com/slevomat/coding-standard/issues/903

Keep up the good work.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/9bbf860dac09845d6fbfdeb69b33961e399e5817

**Comment by ncou** [comment]
thank you !
